parser/LesBonsNumerosAPIParser.go

```diff
@@ -0,0 +1,193 @@
+package parser
```

Comment:
parser -> lesbonsnumeros ?
or
parser -> parser/lesbonsnumeros ?

We need to keep flexibility and abstraction layers. 
eg: if we need to change the API and the parser it will be easy to create a new struct and implement the same methods with the same architecture.